internals/overlord/overlord.go

```diff
@@ -183,6 +194,14 @@ func New(opts *Options) (*Overlord, error) {
 	// before it.
 	o.stateEng.AddManager(o.runner)
 
+	// Load the plan from the Pebble layers directory (which may be missing
+	// or have no layers, resulting in an empty plan), and propagate PlanChanged
+	// notifications to all notification subscribers.
+	err = o.planMgr.Load()
+	if err != nil {
+		return nil, err
```

Comment:
I changed this to: ```cannot load plan: %w```. Would you prefer to expose the concept of layers here ? It may fail on final combined layer validation, for example, so I was thinking perhaps just this is fine ?